### ISSUE TITLE
Go to definition

### ISSUE THREAD

**Issue by Mte90** [mkusher/padawan.vim#35]
I was evaluating to use this kind of features but vim support for tags, but with padawan is not useful create tags only for that so I was wondering if it is possible to do it with padawan itself.

**Comment by mkusher** [comment]
padawan doesn't have this feature right now, you may want to check https://github.com/felixfbecker/php-language-server

**Comment by Mte90** [comment]
Yes I saw that but we have already padawan that scan and index everything so I was wondering why use another server locally only for that feature.

**Comment by mkusher** [comment]
padawan is not maintained, so you might want to choose another completion server. Or you can send PR with goto definition feature :)

**Comment by Mte90** [comment]
Ok if padawan is not maintained is time to switch to something else anyway (https://github.com/padawan-php/padawan.php/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) there are few pr to review anyway
